src/UrlGenerator/BaseUrlGenerator.php

```diff
@@ -1,84 +1,70 @@
 <?php
 
-namespace Spatie\MediaLibrary\UrlGenerator;
+namespace Spatie\Medialibrary\UrlGenerator;
 
 use Illuminate\Contracts\Config\Repository as Config;
-use Spatie\MediaLibrary\Conversion\Conversion;
-use Spatie\MediaLibrary\Models\Media;
-use Spatie\MediaLibrary\PathGenerator\PathGenerator;
+use Illuminate\Contracts\Filesystem\Filesystem;
+use Illuminate\Support\Facades\Storage;
+use Spatie\Medialibrary\Conversion\Conversion;
+use Spatie\Medialibrary\Models\Media;
+use Spatie\Medialibrary\PathGenerator\PathGenerator;
 
 abstract class BaseUrlGenerator implements UrlGenerator
 {
-    /** @var \Spatie\MediaLibrary\Models\Media */
-    protected $media;
+    protected ?Media $media;
 
-    /** @var \Spatie\MediaLibrary\Conversion\Conversion|null */
-    protected $conversion;
+    protected ?Conversion $conversion = null;
 
-    /** @var \Spatie\MediaLibrary\PathGenerator\PathGenerator */
-    protected $pathGenerator;
+    protected ?PathGenerator $pathGenerator;
 
-    /** @var \Illuminate\Contracts\Config\Repository */
-    protected $config;
+    protected Config $config;
 
-    /** @param \Illuminate\Contracts\Config\Repository $config */
     public function __construct(Config $config)
     {
         $this->config = $config;
     }
 
-    /**
-     * @param \Spatie\MediaLibrary\Models\Media $media
-     *
-     * @return \Spatie\MediaLibrary\UrlGenerator\UrlGenerator
-     */
     public function setMedia(Media $media): UrlGenerator
     {
         $this->media = $media;
 
         return $this;
     }
 
-    /**
-     * @param \Spatie\MediaLibrary\Conversion\Conversion $conversion
-     *
-     * @return \Spatie\MediaLibrary\UrlGenerator\UrlGenerator
-     */
     public function setConversion(Conversion $conversion): UrlGenerator
     {
         $this->conversion = $conversion;
 
         return $this;
     }
 
-    /**
-     * @param \Spatie\MediaLibrary\PathGenerator\PathGenerator $pathGenerator
-     *
-     * @return \Spatie\MediaLibrary\UrlGenerator\UrlGenerator
-     */
     public function setPathGenerator(PathGenerator $pathGenerator): UrlGenerator
     {
         $this->pathGenerator = $pathGenerator;
 
         return $this;
     }
 
-    /*
-     * Get the path to the requested file relative to the root of the media directory.
-     */
     public function getPathRelativeToRoot(): string
     {
         if (is_null($this->conversion)) {
             return $this->pathGenerator->getPath($this->media).($this->media->file_name);
         }
 
         return $this->pathGenerator->getPathForConversions($this->media)
-                .$this->conversion->getConversionFile($this->media->file_name);
+                .$this->conversion->getConversionFile($this->media);
     }
 
-    public function rawUrlEncodeFilename(string $path = ''): string
```

Comment:
I believe this removal should be listed in UPGRADING

Reply:
Done